tests/test_services.py

```diff
@@ -580,7 +581,8 @@ def test_ServiceTHREDDS_custom_config(self):
             utils.check_raises(lambda: self.ows.check_request(req), OWSAccessForbidden, msg=msg)
 
         # using unknown prefixes, otherwise allowed file should always be denied
-        unknown_prefixes = ["ncml", "dap4"]  # purposely take normally allowed THREDDS prefixes, validate active config
+        # purposely take normally allowed THREDDS prefixes, validate active config
+        unknown_prefixes = ["ncml", "dap4", "ncss/point"]
```

Comment:
Can you add 2 extra `ncss` and `grid` on their own, just to make sure the prefix matching doesn't incorrectly do any partial match of the pre-split parts?